packages/react/src/components/FuelProvider.tsx

```diff
@@ -0,0 +1,90 @@
+import type { Fuel } from '@fuel-wallet/sdk';
```

Comment:
Can you apply the changes I made here? https://github.com/FuelLabs/fuels-portal/blob/a2c20159bd6b040d53e62d43f02885b38ccb8556/packages/sdk-react/src/components/FuelProvider.tsx

Only the change fuelQueryClient.fetchQuery to fuelQueryClient.invalidateQueries?

This will solve a problem where queries were triggered without generating errors. In case we are not using all the hooks.